snippets/calendar-div.php

```diff
@@ -1,6 +1,9 @@
 <?php
 	$tmpDate = getdate(0);
 	$currentDate = getdate();
+  if (!empty($languageCode)) {
```

Comment:
This should probably be:
```suggestion
  if (!isset($languageCode)) {
```

right?